frontend/src/metabase/static-viz/components/XYChart/utils/values.ts

```diff
@@ -0,0 +1,33 @@
+import { getY } from "./series";
+
+import type { TextProps } from "@visx/text";
+import type { HydratedSeries } from "../types";
+import { measureText } from "metabase/static-viz/lib/text";
+
+// Use the same logic as in https://github.com/metabase/metabase/blob/72d23d2b8e5a1f93b288e5a8738758c9b05d3766/frontend/src/metabase/visualizations/lib/chart_values.js#L318
+export function getValueStep(
+  series: HydratedSeries[],
+  valueFormatter: (value: number) => string,
+  valueProps: Partial<TextProps>,
+  chartWidth: number,
+) {
+  if (series.length === 0) {
+    return 1;
+  }
```

Comment:
Added a guard check so, it won't fail when `series` is an empty array. Unlike the referenced code, combo-chart can display multiple chart at once, making `series` possible to be an empty array.